src/index.ts

```diff
@@ -308,9 +368,35 @@ const oauth2 = <Profiles extends string>({
         token.expires_in = token.expires_in ?? 3600;
         token.created_at = Date.now() / 1000;
 
-        storage.set(req.request, (req.params as TOAuth2Params).name, token);
+        if ((req.params as TOAuth2Params).name === 'twitch') {
```

Comment:
not compatible for everyone

Reply:
see → https://github.com/bogeychan/elysia-oauth2/pull/4#discussion_r1342523516

Reply:
Fixed.